packages/ingest/package.json

```diff
@@ -37,6 +37,7 @@
   "author": "Cumulus Authors",
   "license": "Apache-2.0",
   "dependencies": {
+    "@cumulus/api": "8.1.0",
```

Comment:
It's for 
```
const { createSqsQueues } = require('@cumulus/api/lib/testUtils');
``` 
in the unit test. I could just copy this function. Why do we want to avoid this dependency? Should we just move that function outside of this package?

Reply:
We want to avoid the dependency because `packages/api` is a huge package which itself has a lot of dependencies, which has a lot of undesirable side effects for bootstrap time, compiled code size, or even security vulnerability. Also `packages/api` depends on `packages/ingest`, so you can't really create this dependency, otherwise you now have a circular dependency.

I'd just some manually replicate `createSqsQueues` is doing for the sake of the tests in this package. Also be sure to check whether you really need everything that function is doing